### PR TITLE
Split off CaptureManagerBotModule from HackyAI

### PR DESCRIPTION
**Pull request by reaperrr**
Note: This isn't used in any official mod right know, and known to be bugged on bleed already.
TL;DR: Please prioritize just getting it merged quickly over debugging it.

Dependency for final module PR.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+using OpenRA.Mods.Common.AI;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Manages AI capturing logic.")]
+	public class CaptureManagerBotModuleInfo : ConditionalTraitInfo
+	{
+		[Desc("Actor types that can capture other actors (via `Captures` or `ExternalCaptures`).",
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs]: `ExternalCaptures` doesn't exist anymore. Can you please amend this description?

**Comment by reaperrr** [comment]
Updated.

**Review by pchote — DISMISSED** [review]

File: OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections;
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs]: This is unused.

File: OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.AI;
+using OpenRA.Support;
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Traits/BotModules/CaptureManagerBotModule.cs]: This one as well.

**Review by obrakmann — DISMISSED** [review]
Please remove those two unused usings, then it's good to merge.

**Comment by reaperrr** [comment]
Updated.

**Review by obrakmann — APPROVED** [review]

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/c31b5f223babc51c24ef53737c502056f605f3c3)

**Pull request merged**